### ISSUE TITLE
authn on a link tag?

### ISSUE THREAD

**Issue by RehanSaeed** [hawx/relme-auth#11]
Is it possible to add `authn` to a `<link>` tag in the `<head>` like so?

```html
<link rel="me authn" href="https://github.com/YOU" />
```

All the examples I've seen only do this in anchor tags in the actual HTML body.

**Comment by gRegorLove** [comment]
Yes, this works. I'll bring it up in the microformats community and try to get some updated examples.
